### PR TITLE
execution-comparison: add to RCP

### PR DESCRIPTION
**Pull request by vladarama**
This PR adds the execution-comparison feature to the incubator's RCP.

Signed-off-by: Vlad Arama <vlad.arama@ericsson.com>

### CODE REVIEW

File: rcp/org.eclipse.tracecompass.incubator.rcp/feature.xml
Comment (MatthewKhouzam) [inline rcp/org.eclipse.tracecompass.incubator.rcp/feature.xml:1]: I am curious, why is junit needed for an RCP?

Comment (vladarama) [reply rcp/org.eclipse.tracecompass.incubator.rcp/feature.xml:1]: The build was failing due to a missing requirement. 
Not sure if my change is correct, but it fixed the failing build.

**Review by bhufmann — APPROVED** [review]
LGTM. Thanks!

**Review by MatthewKhouzam — APPROVED** [review]
That's more like it!

**Pull request merged**